tuxemon/resources/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -0,0 +1,81 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
This and other PO files can be generated using a script.  I don't think that we would need comments on each `msgid`/`msgstr` pair, just the slug and translated string would be fine.

Reply:
I thought the comments would be useful, but of course I can remove them :).

Reply:
No need to remove them.  Just wanted to make sure you don't feel obligated to keep them, as it could take a lot of time to make comments for each thing.  Slugs should be descriptive enough so that comments are not needed.  Anyway, comments are fine.